lib/season_stats.rb

```diff
@@ -39,13 +39,14 @@ def games_by_head_coach(season_id)
 
   def winning_percentage_by_head_coach(season_id)
     winning_percentage_by_head_coach = {}
-    games_by_head_coach(season_id).each do |key, value|
-      winning_games = value.find_all { |value| value.result == "WIN" }
+
+    games_by_head_coach(season_id).each do |coach, games|
+      winning_games = games.find_all { |game| game.result == "WIN" }
       if winning_games.length != 0
-        value = ((winning_games.length.to_f / value.length.to_f) * 100).round(2)
-        winning_percentage_by_head_coach[key] = value
+        games = ((winning_games.length.to_f / games.length.to_f) * 100).round(2)
```

Comment:
These smaller average methods are great candidates for our future calculable module